core/src/main/java/jenkins/telemetry/Telemetry.java

```diff
@@ -141,6 +141,22 @@ public static boolean isDisabled() {
         return jenkins == null || !jenkins.isUsageStatisticsCollected();
     }
 
+    /**
+     * Returns true if and only if we are in the time period during which this is supposed to collect data.
+     * @return true iff we're in the time period during which this is supposed to collect data
+     *
+     * @since TODO
```

Comment:
```suggestion
     * @since FIXME
```

(easier to filter these out vs. other random code comments)

Reply:
https://github.com/jenkinsci/jenkins/blob/master/update-since-todo.sh

Reply:
Ah, had not seen that before!